pyspecProcScripts/__init__.py

```diff
@@ -44,7 +43,5 @@
         'zeroth_order_ph',
         'apod_matched_filter',
         'L2G',
-        'masked_mean_multi',
-        'masked_var_multi'
         ]
```

Comment:
let's not spin up new files until we need to. If we're not using these functions in more than one modules, then we don't need to.